blobfuse/src/BlobfuseGcCache.cpp

```diff
@@ -164,6 +164,19 @@ void gc_cache::run_gc_cache()
             {
                 //clean up the file from cache
                 int fd = open(mntPath, O_WRONLY);
+                if (errno == 13) {
+                    close(fd);
```

Comment:
We can add a log here stating this condition has hit and file is write protected. If the chmod fails we will not know whether this code flow was hit or not

Reply:
done